Resources/public/js/models/ez-locationmodel.js

```diff
@@ -98,7 +98,27 @@ YUI.add('ez-locationmodel', function (Y) {
          */
         move: function (options, parentLocationId, callback) {
             options.api.getContentService().moveSubtree(this.get('id'), parentLocationId, callback);
-        }
+        },
+
+        //TODO comment
+        _updatehidden: function (options, hidden, callback) {
+            var locationUpdateStruct = options.api.getContentService().newLocationUpdateStruct();
+
+            locationUpdateStruct.body.LocationUpdate.hidden = hidden;
+            //TODO these should not be mandatory when updating
+            locationUpdateStruct.body.LocationUpdate.sortField = this.get('sortField');
+            locationUpdateStruct.body.LocationUpdate.sortOrder = this.get('sortOrder');
+            options.api.getContentService().updateLocation(this.get('id'), locationUpdateStruct, callback);
+        },
+
+        //TODO comment
+        hide: function (options, callback) {
+            this._updatehidden(options, 'true', callback);
```

Comment:
the REST API expects strings 'true' and 'false' passing a boolean results in '1' and '0' which it doesn't accept.